CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
+# 0.3.0
+
+- chore: remove dependabot
+- feat!: Improve reliability, add unit tests
```

Comment:
Is the ! indicative of a breaking change?

Reply:
yep